### PR TITLE
Drg/patch/sequelize model

### PR DESCRIPTION
**Pull request by drg0drg**
**1. changed the string length in sequelize models to (50) for both User and Exercise models**

  const Exercise = sequelize.define('Exercise', {
    name: {
      type: DataTypes.STRING(50),
      allowNull: false
    },
............etc........ 

  const User = sequelize.define('User', {
    forename: {
      type: DataTypes.STRING(50),
      allowNull: false
    },
    surname: {
      type: DataTypes.STRING(50),
      allowNull: false
    },

NO MODIF ON EMAIL & PASSWORD

**2.Schema now has no exercise table and seeds matches the schema**

### CODE REVIEW

File: models/user.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ const bcrypt = require('bcryptjs');
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
     forename: {
-      type: DataTypes.STRING,
+      type: DataTypes.STRING(50),
```
Comment (Goldenaries658) [inline models/user.js:8]:
```suggestion
      type: DataTypes.STRING(30),
```

File: models/user.js [hunk 2]
```diff
@@ -5,11 +5,11 @@ const bcrypt = require('bcryptjs');
...
       allowNull: false
     },
     surname: {
-      type: DataTypes.STRING,
+      type: DataTypes.STRING(50),
```
Comment (Goldenaries658) [inline models/user.js:12]:
```suggestion
      type: DataTypes.STRING(30),
```

File: db/seeds.sql [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
-VALUES 
-(JohnDoe1, JohnDoe1pass),
-(JohnDoe2, JohnDoe2pass),
-(JohnDoe3, JohnDoe3pass)
+INSERT INTO users
+  (username, password)
+VALUES
+  (JohnDoe1, JohnDoe1pass),
+  (JohnDoe2, JohnDoe2pass),
+  (JohnDoe3, JohnDoe3pass)
```
Comment (Goldenaries658) [inline db/seeds.sql:7]: Add in first/surname

File: db/seeds.sql [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
-
 
+INSERT INTO users_exercises
+  (username, name)
+VALUES
+  (JohnDoe1, exe1),
+  (JohnDoe1, exe3),
+  (JohnDoe2, exe1),
+  (JohnDoe3, exe2),
+  (JohnDoe3, exe3)
```
Comment (Goldenaries658) [inline db/seeds.sql:18]: needs to be exercise_id and an integer

File: db/schema.sql [hunk 5]
```diff
@@ -7,30 +7,21 @@ USE fitness_db;
...
\ No newline at end of file
+
+  --creating the users_exercises table in <fitness_db>
+  CREATE TABLE users_exercises
+  (
+    user_id INTEGER NOT NULL,
+    exercise_id INTEGER NOT NULL,
+    FOREIGN KEY (user_id) REFERENCES users (id),
+    PRIMARY KEY (user_id, exercise_id)
+  )
```
Comment (Goldenaries658) [inline db/schema.sql:27]: This is all indented by the look of it

Comment (drg0drg) [reply db/schema.sql:27]: yes, I know, at save, prettier does some weird things in sql. It indents some of the lines, but not all. I thought to just indent all contents, not to look odd.

**Pull request merged**